devel/python/python/ert_gui/ertwidgets/checklist.py

```diff
@@ -0,0 +1,154 @@
+from PyQt4.QtCore import QSize, Qt
+from PyQt4.QtGui import QToolButton, QHBoxLayout, QLabel, QListWidget, QWidget, QVBoxLayout, QListWidgetItem, QMenu, QAction, QAbstractItemView, QListView
```

Comment:
How long is too long?  I know you're not a fan of 80 column restriction, but 160 is maybe a bit much?